src/kvp/key.rs

```diff
@@ -338,6 +356,14 @@ mod test {
         assert_eq!(key.to_string(), "vendor");
     }
 
+    #[test]
+    fn prefix_equality() {
+        const EXAMPLE_PREFIX_STR: &str = "example.prefix";
```

Comment:
Could we use a "proper" prefix, maybe something like `app.kubernetes.io` or `stackable.tech`.

Reply:
Using a real label prefix in: 5962779